lib/daemon_controller.rb

```diff
@@ -438,7 +432,7 @@ def no_activity?(seconds)
   end
 
   def pid_file_available?
-    File.exist?(@pid_file) && File.stat(@pid_file).size != 0
```

Comment:
`File.empty?` requires Ruby 2.4. We haven't made a decision yet on which Ruby version to support in Passenger, so for now you should use the alternative `File.zero?`.